packages/core/src/lib/history.unit.test.ts

```diff
@@ -24,7 +24,10 @@ vi.mock('./upload', () => ({
 
 describe('history', () => {
   it('should check out all passed commits and reset to initial branch or tag', async () => {
-    await history(MINIMAL_HISTORY_CONFIG_MOCK, ['abc', 'def']);
+    await history(MINIMAL_HISTORY_CONFIG_MOCK as HistoryOptions, [
```

Comment:
❔ Why not create `MINIMAL_HISTORY_CONFIG_MOCK` directly as the  type `HistoryOptions`, so that it doesn't have to be overriden everywhere?

Reply:
Thx! implemented